frontend/src/components/welcome/pricing.js

```diff
@@ -121,30 +121,30 @@ class Pricing extends Component {
             <Grid item key={ tier.title } xs={ 12 } sm={ tier.title === 'Enterprise' ? 12 : 6 } md={ 4 }>
               <Card>
                 <CardHeader
-                  title={ tier.title }
-                  subheader={ tier.subheader }
+                  title={ this.props.intl.formatMessage(messages.tiersTitle)}
+                  subheader={ this.props.intl.formatMessage(messages.tiersSubheader) }
                   titleTypographyProps={ { align: 'center' } }
                   subheaderTypographyProps={ { align: 'center' } }
                   className={ classes.cardHeader }
                 />
                 <CardContent>
                   <div className={ classes.cardPricing }>
                     <Typography variant='h5' color='textPrimary'>
-                      <small>Fee</small> { tier.price }
+                      <small>Fee</small> { this.props.intl.formatMessage(messages.tiersPrice) }
                     </Typography>
                     <Typography variant='subheading' color='textSecondary'>
                       <FormattedMessage id='welcome.pricing.month' defaultMessage=' / issue' />
                     </Typography>
                   </div>
-                  { tier.description.map(line => (
+                  { this.props.intl.formatMessage(messages.tiersDescription).split(',').map(line => (
```

Comment:
@etfaghaoubeid  This is kind of tricky and we should create a key for each item as an array